Turf.xcodeproj/project.pbxproj

```diff
@@ -27,6 +38,9 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		353E9B011F3E08BD007CFA23 /* Turf copy-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "Turf copy-Info.plist"; path = "/Users/fredrik/Workspace/mapbox/turf-swift/Turf copy-Info.plist"; sourceTree = "<absolute>"; };
```

Comment:
Are you sure it’s OK to use an absolute path here?

I’ve found that the same Info.plist can almost always be shared among iOS, macOS, tvOS, and watchOS targets, so it may be unnecessary to even have a Turf copy-Info.plist around.